modules/packages/TOML.chpl

```diff
@@ -165,8 +165,8 @@ module TomlParser {
         if !source.isEmpty() {
           while(readLine(source)) {
             var token = top(source);
-
-            if token == '#' {
+         
```

Comment:
nit: remove trailing white space

Reply:
pro-tip: configure your editor to display trailing white space to avoid this pitfall in the future.